### DISCUSSION TITLE
ui suggestion

### DISCUSSION THREAD

**Discussion by thduc87** [seerge/g-helper#1164] [Ideas]
hi @seerge i'd like to make a suggestion on the ui
![image](https://github.com/seerge/g-helper/assets/132545384/b627a134-4ab9-4e26-b834-a7bb6f4712e0)
is it possible to change the check mark to like a bullet point instead? i personally this bullet point is simpler and matches the overall vibe of your app's ui, what do you think?

**Comment by seerge** [comment]
@thduc87 thats a default UI component, i don’t think that you can change the checkbox there anyhow unfortunately
